packages/s2-core/src/utils/color.ts

```diff
@@ -26,6 +26,21 @@ export const shouldReverseFontColor = (color: string) => {
   return tinycolor(color).getLuminance() <= 0.5;
 };
 
+/**
+ * @param backgroundColor
+ * @param fontColor
+ * @returns 是否满足对比度要求
+ */
+export const isMeetContrastRequirement = (
```

Comment:
```suggestion
export const isReadableText = (
```
`tinycolor` 的这个 `isReadable` 其实就挺表意的